### ISSUE TITLE
Parse data public repo failed

### ISSUE THREAD

**Issue by timboven** [Hacklone/private-bower#178]
I'm running private-bower within docker (used the following container: https://hub.docker.com/r/tandrup/private-bower/) and I also made sure that I upgraded it to the latest version (of private-bower).
Within this container, it's failing to download the public repo metadata due to a parse exception.

I did some small investigations and came to this piece of code:
    function _parsePackage(data) {
        var jsonData;
        try {
            if (data instanceof Array || data instanceof Object) {
              jsonData = data;
            } else {
              jsonData = JSON.parse(data);
            }

When I replaced the whole if by just:
jsonData = data;
 it starts to work - so the returned type of data was something else - but not sure what it was.

**Comment by Hacklone** [comment]
Hey,

This cloud be caused by the public repo url change. Could you try it with the latest version?

Thanks,
Barna

**Comment by reflectivedevelopment** [comment]
I found that changing the repo url from http://bower.herokuapp.com/packages/ to http://bower.herokuapp.com/packages worked for me

**Comment by timboven** [comment]
I created a copy of the docker container and upgraded to the latest version of private-bower and it still wasn't working - but I also tried removing the slash at the end (as also reported by @reflectivedevelopment) and this worked.
I verified with the previous version of the container (by altering the url) and that still didn't work.
So in short - it looks fixed in the latest version, if you also use the updated url.

Kind regards,
Tim
